comeet/UI/MyAgenda/MyAgendaViewController.swift

```diff
@@ -77,8 +76,8 @@ private extension MyAgendaViewController {
     }
     
     func goToSelectedSection() {
```

Comment:
Oh! once you swipe once `pageViewController(,didFinishAnimating:)...` it hides the guide view on the screen you swiped away from.